iac/settings/utils.py

```diff
@@ -0,0 +1,28 @@
+import re
+import subprocess
+
+
+def get_branch_name(strip_punctuation: bool = True) -> str:
+    """
+    Parse the current branch name.
+    """
+    # TODO Looks for branch in ENV first, ie - Github actions env var
```

Comment:
Does this need a card, or is it handled in the CI/CD ticket?

Reply:
I added this to M1700